.github/workflows/storybook-deploy.yml

```
@@ -0,0 +1,32 @@
+name: Deploy Storybook to GitHub Pages
+on: [push]
+jobs:
+  publish:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - uses: pnpm/action-setup@v3
+        name: Install pnpm
+        with:
+          version: 8
+          run_install: false
```

```suggestion
          run_install: true
```
This should let us remove the next step.